src/VirtualClient/VirtualClient.Main/CommandBase.cs

```diff
@@ -292,6 +292,11 @@ protected virtual IServiceCollection InitializeDependencies(string[] args)
                 this.LoggingLevel = LogLevel.Trace;
             }
 
+            if (this.Debug)
```

Comment:
We already have this in, why do you need to add it?

Reply:
Yeah this looks like something came from merge conflict. Will remove it.